ws-android/files/workspace/a04-list-paths.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash  
```

Comment:
This whole file should be part of check-environment by the looks of it. It's just verifying environment configs.

The symlinks should be in the docker container if they're required, there is nothing dynamic in here.